Frameworks/include/BridgeHelpers.h

```diff
@@ -132,7 +132,7 @@ static inline bool shouldUseConcreteClass(Class self, Class base, Class derived)
 
 // Helper macro for implementing allocWithZone
 #define ALLOC_PROTOTYPE_SUBCLASS_WITH_ZONE(NSBridgedType, NSBridgedPrototypeType)                            \
-    (NSObject*) allocWithZone : (NSZone*)zone {                                                              \
+    (NSObject*)allocWithZone : (NSZone*)zone {                                                               \
```

Comment:
would like to +1 this: formatting-only changes like this clutter up git history and make it harder to figure out when a regression might've occurred